### DISCUSSION TITLE
Before I go buy a Radeon, just want to check that is the "happy path" for Regolith (as opposed to NVIDIA)...

### DISCUSSION THREAD

**Discussion by nrvale0** [regolith-linux/regolith-desktop#929] [Q&A] [answered]
I've been running Pop!_OS because, IMO, it best supports NVIDIA GPUs. Even using the System76 packages on Ubuntu or Debian isn't as smooth as it is on Pop itself. The problem is that Pop is itself then a weird corner case for Regolith support...its sort of Ubuntu 22.04 but not really. And Regolith 3 wants Ubuntu 23.04 anyway.

So before I go buy a Radeon so that I can run Ubuntu 23.04 and thus be within the set of target OSes for Regolith 3, can I get an affirm that nothing about R3 expects or requires NVIDIA GPU?

**Comment by cfsmp3** [comment, marked as answer]
I'm running it perfectly on a Framework Laptop 12th gen, nothing nVidia there. It's intel though, not AMD.

**Reply by nrvale0** [reply]
Regolith 3 w/ Sway?

**Comment by cfsmp3** [comment]
> Regolith 3 w/ Sway?



> Regolith 3 w/ Sway?

Yes. I'm the person that brought the person that did all the work on sway to the team :-)

I take the credit as beta tester and nagger on this work.

**Reply by nrvale0** [reply]
Super. Thank you! Off to Radeon Land. :D

**Comment by kgilmer** [comment]
I have also had good experience with Intel and AMD with Regolith on Ubuntu and Debian.  I have returned laptops with nvidia hardware due to poor Linux support.

**Comment by nrvale0** [comment]
FWIW, I am now on Radeon and things are behaving nicely. Tearing on workspace switching is noticeably less. I have yet to make the jump to straight Debian or Ubuntu but perhaps soon.

**Reply by srosato** [reply]
So, just to summarize, you are on Radeon, Regolith 3 PopOS that sits on Ubuntu 23.04? Have you tested gaming?

I am currently using on PopOS with regolith 2 (on Ubuntu 22.04) and borrowed an Nvidia RTX 3070 from a friend and everything runs very well, i3 workspace switching is fast and snappy (I did not add compositor though) I can easily game 1440p ultra settings (tested Rocket League and Baldur's Gate 3).

I am hesitant to buy the RTX from my friend because I might want to try sway or hyprland soon and it seems Radeon has better support. It depends how similar sway feels with i3, because since I switched from AwesomeWM to i3, I have been completely satisfied.

Which Radeon have you bought? I am eyeing the 6800 XT or maybe I should try the 7000 series, but I think to do that I will need to update my OS to 23.04+

**Reply by srosato** [reply]
Just to follow up, I ended up buying a 6800 XT and everything has been very smooth and the gaming experience very good as well on PopOS.

@nrvale0 Thx for your reply below!

**Comment by nrvale0** [comment]
I dropped an RTX 3700 because even with the System76 drivers loaded I was getting a lot of tearing, the occasional lingering video corruption, and massive instability when trying to use regolith's sway variant. I've since rotated to a Radeon 6700 series and had better reliability and less tearing. I have at least been able to get a regolith-sway session to spin up but have been unable to get it to honor my regolith/i3 config in significant ways. So I'm now on Debian with generally more stability for my workstation but still on X11. I'm unsure I will make the Wayland jump soon as I don't see a lot of utility in the effort.

I can't say much about the Radeon gaming experience as I don't really game. The Radeon seems to be handling video playback at fullscreen better than did the RTX.

**Reply by srosato** [reply]
Same for me, seems lots of effort to go on Wayland, sticking with i3 and X11 as of now
